build/parser.mjs

```diff
@@ -869,7 +869,7 @@ class Parser {
   addWeaponWikiaData(item, wikiaItem) {
     item.attacks = wikiaItem.attacks;
     item.ammo = wikiaItem.ammo;
-    item.marketCost = wikiaItem.marketCost;
+    item.marketCost = wikiaItem.marketCost || undefined;
```

Comment:
Could you add a comment here just so we know why exactly we're doing this weird or with undefined? Or make it more clear the intention? Like:
```suggestion
    item.marketCost = wikiaItem.marketCost === '' ? undefined : wikiaItem.marketCost;
```

Reply:
lol, i actually asked him to do it this way. this way if it's ever falsy, we specify exactly the falsy value it should be

Reply:
future proof it sure